lib/pdf_fill/forms/va686c674.rb

```diff
@@ -485,15 +485,22 @@ class Va686c674 < FormBase
                 key: 'veteran.previousMarriage.startCity[%iterator%]',
                 limit: 18,
                 question_num: 14,
-                question_suffix: 'E',
+                question_suffix: 'H',
                 question_text: 'PREVIOUS MARRIAGE HISTORY > CITY'
               },
               'state' => {
                 key: 'veteran_marriage_history.start_location.state[%iterator%]',
                 limit: 2,
                 question_num: 14,
-                question_suffix: 'E',
+                question_suffix: 'I',
                 question_text: 'PREVIOUS MARRIAGE HISTORY > STATE'
+              },
+              'country' => {
+                key: 'veteran_marriage_history.start_location.country[%iterator%]',
+                limit: 2,
+                question_num: 14,
+                question_suffix: 'J',
+                question_text: 'PREVIOUS MARRIAGE HISTORY > COUNTRY'
```

Comment:
Could we (or should we) cover this change with a spec? Same question for the other hash additions in this file.

Reply:
There is a filler_spec that generates 3 versions of the PDF (simple, kitchen_sink, and overflow).  I've updated the fixtures to include some locations with countries outside the US, while leaving some inside the US.  The location for marriage history with country outside the US is here: https://github.com/department-of-veterans-affairs/vets-api/blob/d33be24b41da7cc55f0e864697c38b9c7ac08897/spec/fixtures/pdf_fill/686C-674/overflow.json#L69-L73